editor/editor_properties_array_dict.h

```diff
@@ -72,6 +73,10 @@ class EditorPropertyDictionaryObject : public RefCounted {
 	void set_new_item_key(const Variant &p_new_item);
 	Variant get_new_item_key();
 
+	void update_new_item_key_invalidity();
+	bool get_new_item_key_invalidity();
+	Callable validity_change_callback;
```

Comment:
Should this really be public?

Reply:
I don't how it would work without it being public since it needs the EditorProperty to set the callback.

Reply:
That sounds like something that should use a method, exposing variables directly is not the general appropriate way to handle things in c++